docs/src/imagesegmentation.md

```diff
@@ -0,0 +1,317 @@
+# ImageSegmentation.jl
+
+## Introduction
+
+Image Segmentation is the process of partitioning the image into regions that have similar attributes. Image segmentation has various applications e.g, medical image segmentation, image compression and is used as a preprocessing step in higher level vision tasks like object detection and optical flow. This package is a collection of image segmentation algorithms written in Julia.
+
+## Installation
+
+```julia
+Pkg.add("ImageSegmentation")
+```
+
+## Example
+
+Let's see an example on how to use the segmentation algorithms in this package. We will try to seperate the deer, the ground and the sky in the image below using seeded region growing algorithm. This algorithm initializes the regions as the user-provided seed points and iteratively adds pixels to each region.
+
+![Original](assets/segmentation/deer.jpg) 
+
+```julia
+using Images, ImageSegmentation, ImageView
+
+img = load("deer.jpg")
```

Comment:
Yes, a link is needed to ensure reproducibility.

Can one see a license statement somewhere for the new images? It doesn't happen often, but occasionally one gets asked to prove that you have rights to the image, and if it doesn't come with a license statement (or site policy) then that's not possible. (All the images at TestImages were selected because we could find a statement somewhere.)